core-web/libs/portlets/edit-ema/portlet/src/lib/shared/enums.ts

```diff
@@ -23,5 +23,7 @@ export enum EDITOR_STATE {
 
 export enum EDITOR_MODE {
     EDIT = 'edit',
-    PREVIEW = 'preview'
+    VARIANT = 'variant',
```

Comment:
Not sure that this is a good idea, is missleading, because variant could be also edit.

Reply:
Added 2 states for variants.